simulariumio/file_converter.py

```diff
@@ -82,5 +81,11 @@ def update_trajectory_info_version(self, data: Dict[str, Any]) -> Dict[str, Any]
                 "fovDegrees": DEFAULT_CAMERA_SETTINGS.FOV_DEGREES,
             }
             data["trajectoryInfo"]["version"] = 2
-        print(f"Updated TrajectoryInfo v1 -> v{self.current_trajectory_info_version}")
+        if int(data["trajectoryInfo"]["version"]) == 2:
```

Comment:
is this still `original_version` here?

Reply:
no because if it started as v1, the data should now be at v2 from going through the first loop and the `original_version` is still 1, no longer current. 

Megan was also confused by how I set up this function, but I didn't want to have to write a loop to go from v1 -> v2 and another to go from v1 -> v4 and have to redundantly redo the fields that are added between v1 and v2 in each loop. Is there a better way to write it though?

Reply:
You could pull the converters out as functions. So the first if block could call, ‘convertv1tov2’ and convertv2tov3’, successively on the data.  And the second if else block would just call ‘convertv2tov3’

Reply:
i like that idea

Reply:
ok did that